dipy/tracking/tests/test_life.py

```diff
@@ -162,7 +162,7 @@ def test_fit_data():
     gtab = grad.gradient_table(fbval, fbvec)
     ni_data = nib.load(fdata)
     data = ni_data.get_data()
-    tensor_streamlines, _ = load_tractogram(fstreamlines)
```

Comment:
Could you also remove the import for load_tractogram

Reply:
Indeed, Thank you! Done in db26854